.teamcity/settings.kts

```diff
@@ -388,11 +389,21 @@ object RunAllUnitTests : BuildType({
 				(cd apps/o2-blocks/ && yarn build --output-path="../../artifacts/o2-blocks")
 
 				# Build wpcom-block-editor
-				(cd apps/wpcom-block-editor/ &&  NODE_ENV=development yarn build --output-path="../../artifacts/wpcom-block-editor")
+				cd apps/wpcom-block-editor/
+				NODE_ENV=development yarn build
+				cd dist/
+				tee build_meta.txt <<-EOM
+					commit_hash=%build.vcs.number%
+					commit_url=https://github.com/Automattic/wp-calypso/commit/%build.vcs.number%
+					build_number=%build.number%
+					EOM
+				zip -r ../../../wpcom-block-editor.zip .
+				cd ../../../
 
 				# Build notifications
 				(cd apps/notifications/ && yarn build --output-path="../../artifacts/notifications")
 			"""
+			dockerImage = "registry.a8c.com/calypso/ci-wpcom:latest"
```

Comment:
@scinos, `zip` is not available on the default image, but it is on the wpcom image. Should we change the image here, or should we update the default image to include `zip`?

Reply:
I think `apps/wpcom-block-editor` (and anything under apps) should be built using the image `ci-wpcom`